spec/config_spec.rb

```diff
@@ -35,6 +35,15 @@
     expect(config.dashboard_update_interval).to eq '0 0 * * *'
   end
 
+  it 'sets password for seeds' do
+    expect(config.user_password).to eq 'Str0ngPa$$word12'
+
+    seeds = (Dibber::Seeder.seed(:user, name_method: :email)).count
```

Comment:
Great stuff! Now consider, how using `describe`, `let` and `before` could result in a spec that runs with `-f doc` to better describe what is under test?

Reply:
This test has now been updated to use describe, before and specify